django_freeradius/static/django-freeradius/js/radcheck.js

```diff
@@ -1,8 +1,11 @@
+var django = null;
+
 django.jQuery(function ($) {
     'use strict';
     // stop here if in add mode
+    var gettext = false;
     if (!('.deletelink').length) { return; }
-    if (!gettext) { var gettext = function (text) { return text; }}
+    if (!gettext) { gettext = function (text) { return text; }; }
```

Comment:
since we can't have the `var` statement in the beginning of the function definition, can you add back the `var` statement here or does jslint complain?